tartiflette/language/visitor/visitor.py

```diff
@@ -0,0 +1,98 @@
+from typing import Any, List, Optional, Union
+
+from tartiflette.language.visitor.constants import BREAK, OK, SKIP
+from tartiflette.language.visitor.utils import get_visit_function
+
+
+class Visitor:
+    """
+    Base class for AST visitor.
+    """
+
+
+class ParallelVisitor(Visitor):
```

Comment:
I don't seem to find anything // here, why chose this name ?

Reply:
Actually it was just to fit the naming from the `graphql-js` reference. It doesn't do in // like a `async.gather` would but each node will be visit by each visitor.

If you have a better naming for it I would be glad to rename it 👍

Reply:
"DefaultVisitor" ?

Reply:
Hum... IMO, it does reflect that fact it'll will handle multiple visitor. It does make sense to name it `DefaultVisitor` because our `validate_***` function use it but it's not like its the main visitor that people should use. I think that naming it `DefaultVisitor` could lead to misunderstanding.

What about `DelegateToManyVisitor` or `MultipleVisitor`?

Reply:
MultipleVisitor makes more sens to me then.